server/routes/bookingRequest.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+
+const router = express.Router();
+const protect = require("../middleware/auth");
+const { createBookingRequest, updateBookingRequest, getBookingRequests } = require("../controllers/bookingRequest");
+
+router.route("/create").post(protect, createBookingRequest);
```

Comment:
```suggestion
router.route("/").post(protect, createBookingRequest);
router.route("/:requestId").patch(protect, updateBookingRequest);
router.route("/").get(protect, getBookingRequests);
```

You can simplify these routes :)

Reply:
resolved by 783b90ecbe1678b1b3df6d390464cf96bda4fdfc